shared/oae/css/oae.base.css

```diff
@@ -897,3 +897,31 @@ ul.as-list li.as-result-item.active {
 .oae-markdown-embedded p:first-child {
     margin-top: 0;
 }
+
+/**
+ * Markdown table styles
```

Comment:
This still looks a bit off:

<img width="251" alt="screen shot 2016-06-02 at 11 28 05" src="https://cloud.githubusercontent.com/assets/109850/15756223/1f934740-28b5-11e6-9c64-b1fa39a20a69.png">